root/Makefile

```diff
@@ -122,6 +122,11 @@ coverage:: pre-coverage
 	 WITH_COVERAGE=true GOPROXY=$(GOPROXY) GOPRIVATE=$(GOPRIVATE) ./scripts/shell-wrapper.sh test.sh
 	 go tool cover --html=/tmp/coverage.out
 
+## test-e2e:        run only e2e test (use inside a dev pod)
```

Comment:
Nit: I think this would be easier to understand if it was under `test` instead of right above `e2e`.